mlfinlab/feature_importance/cfi.py

```diff
@@ -0,0 +1,126 @@
+"""
+An Implementation of the Clustered Feature Importance algorithm described by
+Dr Marcos Lopez de Prado in 'Clustered Feature Importance (Presentation Slides)'
+(https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3517595)
+"""
+
+#Imports
+import numpy as np
+import pandas as pd
+import mlfinlab as ml
+from sklearn.metrics import log_loss
+from scipy.cluster.hierarchy import linkage, fcluster
+from mlfinlab.codependence import variation_of_information_score
+
+def get_VI_matrix(df : pd.DataFrame):
```

Comment:
Can we make a universal function which builds dependence matrix using functions from mlfinlab.codependence module?

Reply:
Can you elaborate on what you meant by  universal function?

Reply:
So, as of now in our codependence module we have:
- VI
- Mutual info
- Distance correlation

And simple linear correlation. It would be greate to have a function:

get_dependence_function(algorithm='variation_information') where method is one of algorithms described above or just Pearson correlation

Reply:
Yeah, that would be very handy. I will try to do that.